.github/workflows/trivy.yml

```diff
@@ -0,0 +1,57 @@
+name: Trivy Scanner
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 8 * * *" # runs at 8 AM UTC, 4 AM EST.
+
+jobs:
+  trivy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: Build all projects without running tests
+        run: ./gradlew --build-cache build -x test -x spotlessCheck
+
+      - name: Construct docker image name and tag
+        id: image-name
+        run: |
+          GITHUB_REPO=$(basename ${{ github.repository }})
+          GIT_SHORT_HASH=$(git rev-parse --short HEAD)
+          echo ::set-output name=name::${GITHUB_REPO}:${GIT_SHORT_HASH}
```

Comment:
I guess it doesn't hurt to do this but since the image isn't pushed to artifactory I don't think it matters what the image is called. You could use a fixed name for both the build output and the scanner input.

Reply:
I'm okay with keeping this as is to be consistent with our other pipelines.